fs/fs.go

```diff
@@ -359,8 +359,8 @@ func dockerDMDevice(driverStatus string) (string, uint, uint, uint, error) {
 	)
 
 	_, err = fmt.Fscanf(bytes.NewReader(dmTable),
-		"%d %d %s %d:%d %d:%d %d %d %d %s",
-		&bkt, &bkt, &bkts, &bkt, &bkt, &major, &minor, &dataBlkSize, &bkt, &bkt, &bkts)
+		"%d %d %s %d:%d %d:%d %d",
```

Comment:
Is there any kernel documentation or paths you can link to here to make this easier to verify, and easier to understand/modify in the future?

Same below.